pkg/sfu/downtrack.go

```diff
@@ -588,7 +586,8 @@ func (d *DownTrack) Bind(t webrtc.TrackLocalContext) (webrtc.RTPCodecParameters,
 		d.setBindStateLocked(bindStateBound)
 		d.bindLock.Unlock()
 
-		d.forwarder.DetermineCodec(codec.RTPCodecCapability, d.Receiver().HeaderExtensions())
+		receiver := d.Receiver()
+		d.forwarder.DetermineCodec(codec.RTPCodecCapability, receiver.HeaderExtensions(), receiver.IsSimulcast())
```

Comment:
nit: we can directly call d.Receiver.IsSimulcast()

Reply:
Split it out as it was also getting `HeaderExtensions` from receiver

Reply:
oops yea, sorry I missed it.